app/styles/_variables_mixins.scss

```diff
@@ -31,6 +32,7 @@ $sidebar_icon_lighter:  $navy_light;
 $sidebar_sublink_text:  #c2d2e3;
 $view_sub_nav:          #6784a2;
 $content_border:        $blue_lightest;
+$brown:                 #711214;
```

Comment:
I couldn't see in the screenshots where this color gets applied (and am unable to checkout the branch right now). Can you post a screencap of where this brown is used specifically please?